core/src/main/java/io/questdb/std/str/Utf8SplitString.java

```diff
@@ -112,7 +111,7 @@ public String toString() {
 
     @Override
     public long zeroPaddedSixPrefix() {
-        return Unsafe.getUnsafe().getLong(auxLo) & VARCHAR_INLINED_PREFIX_MASK;
```

Comment:
Shouldn't we keep the masking? The upper 2 bytes contain the offset, not the prefix.